aeolis/console_typer.py

```diff
@@ -0,0 +1,150 @@
+'''This file is part of AeoLiS.
+
+AeoLiS is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+AeoLiS is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with AeoLiS.  If not, see <http://www.gnu.org/licenses/>.
+
+AeoLiS  Copyright (C) 2023 AeoLiS Development Team
+
+Delft University of Technology
+Faculty of Civil Engineering and Geosciences
+Stevinweg 1
+2628CN Delft
+The Netherlands
+
+'''
+
+from __future__ import absolute_import, division
+
+import os
+import logging
+import shutil
+import numpy as np
+import aeolis as aeolis_pkg
+from aeolis.model import AeoLiSRunner, WindGenerator
+
+import typer
+from typing import Annotated, Optional
```

Comment:
This needs Python>=3.9

Testing the update CLI triggers the following error on Python 3.8.
```
$ aeolis-new --help
Traceback (most recent call last):
  File "/../venvs/../bin/aeolis-new", line 5, in <module>
	from aeolis.console_typer import app
  File "/../venvs/../lib/python3.8/site-packages/aeolis/console_typer.py", line 36, in <module>
	from typing import Annotated, Optional
ImportError: cannot import name 'Annotated' from 'typing' (/usr/lib/python3.8/typing.py)
```
[Annotated is introduced in Python 3.9](https://docs.python.org/3/whatsnew/3.9.html#typing)

Changes work fine on Python 3.9